sed/config/NXmpes_config-HEXTOF.json

```diff
@@ -137,20 +130,18 @@
     "frequency/@units": "@attrs:metadata/scientificMetadata/Source/repetition_rate/unit",
     "associated_beam": "/entry/instrument/beam_probe"
   },
-  "/ENTRY[entry]/INSTRUMENT[instrument]/beam_TYPE[beam_probe]": {
+  "/ENTRY[entry]/INSTRUMENT[instrument]/beamTYPE[beam_probe]": {
     "distance": 0.0,
     "distance/@units": "mm",
     "incident_energy": "@attrs:metadata/scientificMetadata/Source/photon_energy/value",
-    "incident_energy/@units": "@attrs:metadata/scientificMetadata/Source/dispersion/unit",
-    "incident_energy_dispersion": "@attrs:metadata/scientificMetadata/Source/dispersion/value",
-    "incident_energy_dispersion/@units": "@attrs:metadata/scientificMetadata/Source/dispersion/unit",
-    "pulse_duration": "t.b.d.",
+    "incident_energy/@units": "@attrs:metadata/scientificMetadata/Source/photon_energy/unit",
+    "pulse_duration": 100,
     "pulse_duration/@units": "fs",
```

Comment:
Can be also removed here if unknown.

Reply:
This is a bit unfortunate - this value comes from LOLA measurements but quite often this device is broken and not delivering any data :(

Reply:
Then enter manually in such a case?